raft.go

```diff
@@ -20,6 +20,34 @@ var (
 	keyLastVoteCand = []byte("LastVoteCand")
 )
 
+// getRPCHeader returns an initialized RPCHeader struct for the given
+// Raft instance. This structure is sent along with RPC requests and
+// responses.
+func (r *Raft) getRPCHeader() RPCHeader {
+	return RPCHeader{
+		ProtocolVersion: r.conf.ProtocolVersion,
+	}
+}
+
+// dispositionRPC houses logic about whether this instance of Raft can process
+// an RPC message with the given header.
+func (r *Raft) dispositionRPC(header RPCHeader) bool {
```

Comment:
What does the name "dispositionRPC" mean? How about `func (r* Raft) checkRPCHeader(header RPCHeader) error`, which we could later extend for cluster ID?